src/content/guides/typescript.md

```diff
@@ -90,6 +91,26 @@ module.exports = {
 
 This will direct webpack to _enter_ through `./index.ts`, _load_ all `.ts` and `.tsx` files through the `ts-loader`, and _output_ a `bundle.js` file in our current directory.
 
+Now let's change the import of `lodash` in our `./index.ts` due to the fact that there is no default export present in `lodash` definitions.
```

Comment:
```suggestion
Now lets change the import of `lodash` in our `./index.ts` due to the fact that there is no default export present in `lodash` definitions.
```
https://www.grammarly.com/blog/lets-vs-lets/

Reply:
This is a great catch

Reply:
oh man how many time did we get caught on this :D